.gitignore

```diff
@@ -38,6 +38,7 @@ test/truffle/cexts/**/mkmf.log
 
 # Maven
 lib/truffleruby.jar
+target
```

Comment:
It would probably be more consistent to have a leading`/` in those cases with `/` in the middle as well as it causes no harm.